src/pages/documentation/platform-assurance-sdk/tabs/api-reference.md

```diff
@@ -72,24 +73,26 @@ static Future<String> get extensionVersion async
 assuranceVersion = await FlutterAssurance.extensionVersion;
 ``` --->
 
-<Variant platform="android" api="start-session" repeat="10"/>
+<Variant platform="android" api="start-session" repeat="9"/>
```

Comment:
Just to double check - have you checked on localhost that this properly renders? I'm not 100% sure if this needs to be reduced to a 9, that's why :p